app/weather_api/services/met_service.py

```diff
@@ -0,0 +1,43 @@
+import aiohttp
+import requests
```

Comment:
Why are you using both aiohttp and requests? Preferable you should only use aiohttp (or another async-capable library, like httpx), as requests is not async